src/config_db/Dockerfile

```diff
@@ -1,5 +1,8 @@
 
-FROM skaorca/pytango_ska_dev:latest
```

Comment:
What's the purpose of these Dockerfile changes? I don't understand the benefit of the change, which is probably down to me not understanding Docker very well.

Reply:
Mostly boils down to removing Tango, because workflows don't need it. I also wanted to go to a "slim" version of it, which only mostly works.